drivers/net/wireless/mediatek/mt76/mt7921/debugfs.c

```diff
@@ -236,6 +236,19 @@ mt7921_pm_set(void *data, u64 val)
 	if (val == pm->enable)
 		return 0;
 
+	if (val == 100) {
+		mt7921_mutex_acquire(dev);
+		mt76_connac_mcu_set_deep_sleep(&dev->mt76, 0);
+		mt7921_mutex_release(dev);
+		return 0;
+	}
+	else if(val == 101) {
+		mt7921_mutex_acquire(dev);
+		mt76_connac_mcu_set_deep_sleep(&dev->mt76, 1);
+		mt7921_mutex_release(dev);
+		return 0;
+	}
+
```

Comment:
@objelf I would prefer a dedicated debugs knob for enabling/disabling deep sleep avoid magic numbers

Reply:
ack. i will create dedicated debugs knob for enabling and disabling deep sleep.